week-1/Homework/mandatory/5-magic-8-ball.js

```diff
@@ -45,22 +45,58 @@ Very doubtful.
 
 // This should log "The ball has shaken!"
 // and return the answer.
-function shakeBall() {}
+
+function shakeBall() {
+  let possibleAnswer = [
```

Comment:
This solution works really well, nicely done!

A couple of things to think about (some of which we'll learn more about in week three) which could make this code even easier to work with in the future:
This long list combines a few different types of response together. It may be useful to add a comment at the start of each section, i.e.
```javascript
let allAnswers = [
        // Very positive
        'It is certain.',
	'It is decidedly so.',
	'Without a doubt.',
	'Yes - definitely.',
	'You may rely on it.',

        // positive
	'As I see it, yes.',
	'Most likely.',
	'Outlook good.',
	'Yes.',
	'Signs point to yes.',

        // negative
	'Reply hazy, try again.',
	'Ask again later.',
	'Better not tell you now.',
	'Cannot predict now.',
	'Concentrate and ask again.',

        // very negative
	"Don't count on it.",
	'My reply is no.',
	'My sources say no.',
	'Outlook not so good.',
	'Very doubtful.'
];
```
so that it's easy to see which responses are which.

Even then, though, if you add another answer, you need to change several pieces of code (the list, and the indexes in each if statement). In week 3 we'll learn more about arrays, and about objects, which could make it even easier to work with. Have a think about how they could help in this problem so that adding a new answer only needs one code change when we're learning about them!